steamcmd_servers/ror2/egg-risk-of-rain2.json

```diff
@@ -0,0 +1,31 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-04-17T12:39:54-07:00",
+    "name": "Risk of Rain 2",
+    "author": "alex.chang-lam@protonmail.com",
+    "description": "Risk of Rain 2 dedicated server.",
+    "features": null,
+    "images": [
+        "quay.io\/parkervcp\/pterodactyl-images:debian_wine-5"
+    ],
+    "file_denylist": [],
+    "startup": "xvfb-run wine .\/Risk of Rain 2.exe",
```

Comment:
fails to start the server.

```
:/home/container$ xvfb-run wine ./Risk of Rain 2.exe
wine: cannot find L"./Risk"
```

Reply:
Fixed. Edited start command didn't make it back to the nest